app/views/recipes/index.html.erb

```diff
@@ -3,8 +3,10 @@
   <% @recipe.each do |recipe|%>
     <div>
       <div>
-        <h3><%=recipe.name%></h3>
-        <%=button_to 'REMOVE', "/users/#{current_user.id}/recipes/#{recipe.id}", :method =>:delete, :data=>{ turbo: false } %>
+        <h3><%= link_to recipe.name, user_recipe_path(current_user.id, recipe) %></h3>
+        <% if can? :destroy, recipe %>
+          <%=button_to 'REMOVE', "/users/#{current_user.id}/recipes/#{recipe.id}", :method =>:delete, :data=>{ turbo: false } %>
+        <% end %>
```

Comment:
I think it would also be nice if a button to add another recipe is implemented in addition to the remove.